packages/generator/tests/acceptance/test_client_generator.py

```diff
@@ -22,41 +25,40 @@ def test___command_line_args___create_client___render_without_error(
     golden_path = test_assets_directory / "example_renders" / "measurement_plugin_client"
     filename = f"{module_name}.py"
 
-    with pytest.raises(SystemExit) as exc_info:
-        create_client(
-            [
-                "--measurement-service-class",
-                "ni.tests.NonStreamingDataMeasurement_Python",
-                "--module-name",
-                module_name,
-                "--class-name",
-                "NonStreamingDataMeasurementClient",
-                "--directory-out",
-                temp_directory,
-            ]
-        )
-
-    assert not exc_info.value.code
+    result: Result = create_client(
```

Comment:
ditto

Reply:
removed the similar type hints throughout the file.